2017 hero section

darrendub opened the pull request:
## Purpose
Adding styles, content, and layout to johnsonfamilycookieparty.com to reflect event details and information. 

## Notes
• Since this will not be affecting index.html—only affect stranger.html which isn't linked—I think we can safely merge to master so we can start sharing this with the event planners to get copy and layout approval
• Looking for guidance on responsive styles but that can be in a follow up PR

![screencapture-file-users-dwong-public-johnsonfamilycookieparty-stranger-html-1513324394474](https://user-images.githubusercontent.com/4184008/34032295-40fe4676-e12a-11e7-8431-e4674eb767fd.jpg)

![_users_dwong_public_johnsonfamilycookieparty_stranger html iphone 6 plus](https://user-images.githubusercontent.com/4184008/34032270-20dbd552-e12a-11e7-9e66-4b1e6658c196.png)

emilypstewart committed:
Add .gitignore

emilypstewart committed:
Add scaffolding for manifest

emilypstewart committed:
Add base styles

emilypstewart committed:
Stub out structure

emilypstewart committed:
[Compiled] CSS

darrendub committed:
Merge remote-tracking branch 'origin/2017-hero-section'

darrendub committed:
Adding colors

darrendub committed:
New Logo who dis

Remove depreciated images and update the red color

darrendub committed:
Color names, and type names

emilypstewart committed:
Move assets into images directory

We’ll probably need to compress these.

emilypstewart committed:
Set up asset path

emilypstewart committed:
Add Netflix logo

emilypstewart committed:
Expand button component

emilypstewart committed:
Expand type styles

emilypstewart committed:
Add resemblance of sections

emilypstewart committed:
Flesh out watch button…😬

Not sure about structure

emilypstewart committed:
Add scaffolding for Arcade section

emilypstewart committed:
Tweaks to class names + white space

emilypstewart committed:
[Compiled]

emilypstewart committed:
Remove unused vars

emilypstewart committed:
Add placeholder for type sizes

darrendub committed:
Add content, assets, and copy

Add all the images, fonts, text into each section. Now we need to style
them bad boyz.

darrendub committed:
Merge branch 'master' into 2017-hero-section

darrendub committed:
Clean clean clean

Stranger section: made the layout a bit more balanced
Leaderboard section: put the lists into a grid with a subtle animation
Adios section: added portal with hover animation ty netflix

darrendub committed:
Starting some small screen clean up

darrendub committed:
First pass at responsive + updated content

darrendub committed:
add background color to hero (to avoid flashing on load)

darrendub committed:
Add links to buttons, doh!

emilypstewart commented on assets/stylesheets/_sections.scss:
```diff
@@ -21,15 +26,21 @@
   margin: 0 auto;
   margin-bottom: 4em;
+  width:100%;
+  max-width: 600px;
```
🤔 is this implementation-specific? if not, would it make sense to extract this to `$width--med`? (`$wrapper--med`, etc…?)

emilypstewart commented on assets/stylesheets/_sections.scss:
```diff
@@ -57,13 +68,18 @@
+  background-position: bottom;
   background-repeat: no-repeat;
-  background-size: auto 150%;
+  @media screen and (min-width: $bp-small){
```
what's `$bp`?

emilypstewart replied:
oh, breakpoint. derp.

darrendub replied:
derrrrpy (i totally lifted that variable name from the internet)

emilypstewart commented on assets/stylesheets/_sections.scss:
```diff
@@ -88,16 +103,31 @@
-.leaderboard {
+.leaderboard { // Need to rearrange grid for small screens
+  padding-bottom: 3em;
+  display: grid;
```
Look at you!!

emilypstewart commented on assets/stylesheets/_sections.scss:
```diff
@@ -110,61 +140,160 @@
+  -webkit-animation: blink linear 1.25s infinite;
+  animation: blink linear 1.25s infinite;
+}
+@-webkit-keyframes blink {
```
Nice! Might make sense to separate the keyframes into an animations partial? (unless that seems overkill?)

darrendub replied:
yes maam

emilypstewart commented on assets/stylesheets/_sections.scss:
```diff
@@ -110,61 +140,160 @@
-  align-items: center;
-  display: flex;
+.events-section{
+  background-color: #020710;
```
(i see this raw hex 😜)

darrendub replied:
oh ya hahah i should probably update that lol $variables-only

emilypstewart commented on assets/stylesheets/_sections.scss:
```diff
@@ -110,61 +140,160 @@
+  background-size: auto 100%;
+  width: 320px;
+  height: 320px;
+  @media screen and (min-width: $bp-small+50){
```
the inline media queries are interesting. 🤔 I know @miketlawrence advocates for this…I'm still on the fence, but it works here.

darrendub replied:
i find them easier to pinpoint when im trying to affect the responsive style of a certain class

emilypstewart commented on assets/stylesheets/_sections.scss:
```diff
@@ -110,61 +140,160 @@
+  max-width: 200px;
+}
+
+.upside-up {
```
hahah, only on a Stranger Things project would this name make sense

darrendub replied:
lol

emilypstewart reviewed (approved):
This is awesome! thanks for the clean commit messages; it makes it so much easier to review. Added a few suggestions…but nicely done 🙌

darrendub committed:
Code Clean Up

• Moved animations to a partial
• Converted a hex to a variable
• Deleted commented out styles
